docs/sphinx/getting_started/manipulation.rst

```diff
@@ -10,20 +10,27 @@ The code reported below is taken from the example
 It shows the following functionalities:
 
 - Download models from `Ignition Fuel <https://app.ignitionrobotics.org/dashboard>`_.
-- Insert models during runtime.
+- Insert and remove models during runtime.
 - Exploit the high-level :py:class:`~gym_ignition_environments.models.panda.Panda` helper class to insert the manipulator.
 - Enable custom C++ controllers (:cpp:class:`scenario::controllers::ComputedTorqueFixedBase`).
 - Detect contacts and read contact wrenches.
-- Integrate with :py:class:`~gym_ignition.rbd.idyntree.inverse_kinematics_nlp.InverseKinematicsNLP`.
+- Exploit :py:class:`~gym_ignition.rbd.idyntree.inverse_kinematics_nlp.InverseKinematicsNLP`.
 
-.. figure:: https://user-images.githubusercontent.com/469199/99263406-6f387400-281f-11eb-9a7d-3d6f58681499.png
+.. figure:: https://user-images.githubusercontent.com/469199/99905096-f29a0f80-2cce-11eb-9f1c-002f6c887bc6.png
    :align: center
 
 .. tip::
 
    This example can be the starting point to develop manipulation environments for robot learning.
    Visit the :ref:`getting_started_gym_ignition` page for more information about how to wrap this code in the resources provided by ``gym_ignition``.
 
+.. note::
+
+   This is an illustrative example. A compliant joint controller is used to move the manipulator, it does not perform any
```

Comment:
Nitpick: "compliant joint controller" sounds a bit strange, it is like describe a position controller as a "positional controller".

Reply:
Mmh I'm not sure I got your point. Let's take two examples: pid controller and this computed torque controller. Both are position controllers, but I would not use the "compliant" adjective for a pid controller.

Reply:
Searching a bit more it seems that "compliant control" is indeed in widespread use, please ignore my previous comment.